app/views/content/welcome/yvonnes-story/_article.html.erb

```diff
@@ -0,0 +1,29 @@
+<article class="case-study">
+  <p>
+    Yvonne is and English teacher in London. She uses her love of rap music to
```

Comment:
Should we make this intro (which is in the third person) look different from the rest of the article, which is in the first person? Or put quotes round the first person section?